lib/sources/jenkins.js

```diff
@@ -241,6 +241,9 @@ export default class JenkinsSource extends Source {
       packages.push(pkg);
     }
 
-    return packages;
+    return {
+      packages: packages,
```

Comment:
This can just be
```suggestion
      packages
```

When they key and value object share the same name, you can just use it once.